### ISSUE TITLE
Probably invalid warning by no-js-extension rule

### ISSUE THREAD

**Issue by obrejla** [cvisco/eslint-plugin-requirejs#68]
Not sure whether this should be marked as a warning? Maybe it makes sense to check only AmdRequires? Not sure, what do you think?

``` js
module.exports = function(config) {
    // loads standard karma setup
    var getKarmaSetup = require('./karma.setup.js');
};
```

**Comment by cvisco** [comment]
That should definitely be a warning. Personally, I use the CommonJS style almost exclusively, and would expect this to warn.

**Comment by obrejla** [comment]
I'm not sure, but I saw it in a codebase so I was just curisous. And what should be the "no warning" version of that?

**Comment by cvisco** [comment]
I don't quite follow...but I assume it would be

``` js
module.exports = function(config) {
    // loads standard karma setup
    var getKarmaSetup = require('./karma.setup');
};
```

**Comment by obrejla** [comment]
Fair enough. I suggested to rewrite it without .js and it was accepted ;)
